README.md

```diff
@@ -13,7 +13,23 @@ Please see [action.yml](action.yml)
 
 ## Example Usage
 
-Place this file in ".github/workflows/action.yml". 
+You need two things: 
+ 1. an "ECHIDA" file
```

Comment:
```suggestion
 1. an "ECHIDNA" file
```

Reply:
Um! That was test, just to see if you were paying attention... you passed.